ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivacyController.java

```diff
@@ -173,4 +187,49 @@ public long getSenderNonce(final Address sender, final String privacyGroupId) {
             // private state does not exist
             Account.DEFAULT_NONCE);
   }
+
+  private SendRequest createSendRequest(final PrivateTransaction privateTransaction) {
+    final BytesValueRLPOutput bvrlp = new BytesValueRLPOutput();
+    privateTransaction.writeTo(bvrlp);
+    final String payload = BytesValues.asBase64String(bvrlp.encoded());
+
+    if (privateTransaction.getPrivacyGroupId().isPresent()) {
+      return new SendRequestBesu(
+          payload,
+          enclavePublicKey,
+          BytesValues.asBase64String(privateTransaction.getPrivacyGroupId().get()));
+    } else {
+      final List<String> privateFor =
+          privateTransaction.getPrivateFor().get().stream()
+              .map(BytesValues::asBase64String)
+              .collect(Collectors.toList());
+
+      // FIXME: orion should accept empty privateFor
```

Comment:
Not a fan of leaving TODOs or FIXME comments in the codebase, delete it?

Reply:
What's out position on FIXME vs TODO? I thought FIXMEs have gotta go - but TODOs can be useful ... i.e. identifying "boyscouting" work which you don't want to do now ... suppose you could argue that it should be a Tech-debt JI ...

Reply:
> should be a Tech-debt JI 
That one.

For me it's about our processes around visibility of work that needs doing. 
Few people (if any) scour the codebase for FIXMEs and TODOs to prioritise them in their workload, also then there is having work tickets in two places (the code and JIs).
Keeping it simple, put it a JI ...if it doesn't warrant a JI, then it's a small change i.e. do it right away or don't it at all.